README.md

```diff
@@ -1 +1,31 @@
-# hw01
\ No newline at end of file
+---
+title: "Yunshu Hu"
+---
+
+# hw01
+
+## Basic Info
+
+My name is Yunshu Hu, I am currently back home in Jinan, China. 
+
+I am a rising third year undergraduate student majoring in economics.
+
+Here is the most recent picture of me I can find in my phone. It is taken by my best friend.
+
+![A picture of me](Yunshu_pic.jpg)
+
+## Fun Facts
+
+Here are some fun facts about me:
+
+* I love museums. Wherever I travel, I visit the local museums. My favorite museums so far are:
```

Comment:
The formatting of the list here is really great - I like how you layered lots of elements (i.e. links, emphasis, etc)